### DISCUSSION TITLE
📝【LeetCode】78、90 全组合（子集）

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#75] [Announcements]
https://imageslr.github.io/2020/01/29/leetcode-78.html 

 全组合 问题描述 这道题是 LeetCode 78 题 - 子集。

**Comment by imageslr** [comment]
@TubaVADE Yeah. Thanks for sharing. I found that it's just like my [solution 3](https://imageslr.com/2020/01/29/leetcode-78.html#%E8%A7%A3%E6%B3%95%E4%B8%89%E5%9B%9E%E6%BA%AF), but more clear!

**Comment by fireflyoo** [comment]
我翻译了成了Ruby和Elixir两种语言..  
感觉自己在研究回字有几种写法..  

[78题Ruby版-五种解法](https://fireflyoo.github.io/posts/leetcode-subsets/)  
[78题Elixir版-四种解法](https://fireflyoo.github.io/posts/leetcode-subsets-elixir/)
